cpp/include/Ice/OutgoingResponse.h

```diff
@@ -99,8 +99,8 @@ namespace Ice
 
         /**
          * Get the full details of the exception marshaled into the response.
-         * @return The exception details. For Ice exceptions, it's usually produced by streaming the exception into
-         * a std::ostringstream with operator<<. It's empty when replyStatus() is ReplyStatus::Ok.
+         * @return The exception details. For Ice exceptions, it's produced by streaming the exception into a
+         * std::ostringstream with operator<<. It's empty when replyStatus() is ReplyStatus::Ok.
```

Comment:
I know that you were just (correctly) removing `usually` here... but the `a` isn't correct here. It should be:
```suggestion
         * @return The exception details. For Ice exceptions, it's produced by streaming the exception into an
         * std::ostringstream with operator<<. It's empty when replyStatus() is ReplyStatus::Ok.
```
Because we're pronouncing the _letter_ s which starts with a vowel (it's pronounced like `es`).